build/php.m4

```diff
@@ -1815,6 +1815,7 @@ AC_DEFUN([PHP_PROG_BISON], [
     done
 
     if test "$php_bison_check" != "invalid"; then
+      AC_SUBST([YFLAGS], [-Wempty])
```

Comment:
Per the bison docs the option is called `-Wempty-rule`, not `-Wempty`. Which is right?

Reply:
Abbreviations are recognized.  So `-Wempty-rule` is better. I'll fix that, thanks.